acceptance-tests/run-tests

```diff
@@ -18,12 +18,17 @@ setArgsOverrides(
     .option('accountId', {
       alias: ['p', 'portalId', 'a'],
       type: 'string',
-      description: 'Account ID',
+      description: 'The account ID to use',
     })
     .option('cliPath', {
       alias: 'c',
       type: 'string',
-      description: 'CLI path',
+      description: 'The path to the CLI',
+    })
+    .option('cliNPMVersion', {
```

Comment:
Open to suggestions for a better arg name

Reply:
I feel like just calling it `cliVersion` might be clearer? `cliNPMVersion` kind of makes it sound like its the version of `npm` you're going to use. But don't feel super strongly